content/en/docs/contribute/style/style-guide.md

```diff
@@ -187,6 +183,38 @@ Set the value of `image` to nginx:1.16. | Set the value of `image` to `nginx:1.1
 Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
 {{< /table >}}
 
+## Referring to Kubernetes API resources
+
+This section talks about how we reference API resources in the documentation.
+
+### Clarification about "resource"
+
+Kubernetes uses the word "resource" to refer to API resources, such as `pod`, `deployment`, and so on. We also use "resource" to talk about CPU and memory requests and limits.
```

Comment:
I think it's worth adding one more line here directing a contributor to always refer to API resources as "API resources" to avoid confusion with CPU and memory resources

Reply:
Gotcha